st2client/st2client/client.py

```diff
@@ -73,55 +73,79 @@ def __init__(self, base_url=None, auth_url=None, api_url=None, api_version=None,
 
         # Instantiate resource managers and assign appropriate API endpoint.
         self.managers = dict()
-        self.managers['Token'] = models.ResourceManager(
-            models.Token, self.endpoints['auth'], cacert=self.cacert, debug=self.debug)
-        self.managers['RunnerType'] = models.ResourceManager(
-            models.RunnerType, self.endpoints['api'], cacert=self.cacert, debug=self.debug)
+
```

Comment:
Plus you are gonna see merge conflict with https://github.com/StackStorm/st2/pull/1480 :P.